extensions/wikia/WallNotifications/WallNotificationsEveryone.class.php

```diff
@@ -158,37 +210,34 @@ public function getEntityProcessed( $userId, $entityKey ) {
 			return true;
 		}
 
-		$row = $this->getDB( false )->selectRow( 'wall_notification_queue_processed',
-			[ 'count(*) as cnt' ],
+		$cnt = $this->getDB( false )->selectField( self::WALL_NOTIFICATIONS_QUEUE_PROCESSED_TABLE,
+			'count(*)',
 			[
 				'user_id' => $userId,
 				'entity_key' => $entityKey
 			],
 			__METHOD__
 		);
 
-		if ( $row->cnt == 0 ) {
-			wfProfileOut( __METHOD__ );
-			return false;
-		}
-
-		$this->setEntityProcessed( $userId, $entityKey );
 		wfProfileOut( __METHOD__ );
-		return true;
+
+		// there's at least a single matching row - entity has already been proccessed
+		// .e. there's an entry in wall_notification table
+		return ( $cnt > 0 );
 	}
 
-	public function setQueueProcessed( $userId ) {
+	private function setQueueProcessed( $userId ) {
 		global $wgMemc;
 
 		wfProfileIn( __METHOD__ );
 
 		$cacheKey = $this->getQueueProcessedCacheKey( $userId );
-		$wgMemc->set( $cacheKey, true );
+		$wgMemc->set( $cacheKey, true ); // cache for infite time - memcache key contains a cache buster value
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)